### ISSUE TITLE
Performance benchmark

### ISSUE THREAD

**Issue by JpEncausse** [transcranial/keras-js#17]
Hi,
Very very impressive library, hope to let it run on a NodeJS server.

Did you perform some benchmark to compare the library to Tensorflor / Caffe app ? I assume it's slower but I wondering how much ?

**Comment by transcranial** [comment]
@JpEncausse Indeed, it's quite a bit slower than raw tensorflow right now--there are a number of places yet to be optimized. But yes, benchmarks would be a good thing to have.
